providers/dns/googledomains/googledomains.go

```diff
@@ -0,0 +1,133 @@
+package googledomains
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/go-acme/lego/v4/challenge"
+	"github.com/go-acme/lego/v4/challenge/dns01"
+	"github.com/go-acme/lego/v4/platform/config/env"
+	"google.golang.org/api/acmedns/v1"
+	"google.golang.org/api/option"
+)
+
+// Environment variables names.
+const (
+	envNamespace = "GOOGLE_DOMAINS_"
+
+	EnvAccessToken = envNamespace + "ACCESS_TOKEN"
+	EnvZoneName    = envNamespace + "ZONE_NAME"
```

Comment:
This constant is not used, is it expected?

Reply:
Right, I ended up not using this oops.